src/main/java/org/gradle/playframework/sourcesets/LessSourceSet.java

```diff
@@ -0,0 +1,38 @@
+package org.gradle.playframework.sourcesets;
+
+import org.gradle.api.Action;
+import org.gradle.api.file.SourceDirectorySet;
+
+/**
+ * Represents a source set containing LESS sources.
+ * <p>
+ * The following example demonstrate the use of the source set in a build script using the Groovy DSL:
+ * <pre>
+ * sourceSets {
+ *     main {
+ *         less {
+ *             srcDir 'app/assets'
+ *             include '{@literal **}/*.less'
+ *             exclude '{@literal **}/_*.less'
+ *         }
+ *     }
+ * }
+ * </pre>
+ */
+public interface LessSourceSet {
```

Comment:
Out of curiosity, does this become accessible via the Kotlin DSL as well?

Reply:
Honestly I am not using Kotlin DSL so I cannot verify. But I believe it should because it follows exactly the same pattern as the other source sets in the existing codebase.